compiler/ballerina-lang/src/main/java/io/ballerina/projects/Package.java

```diff
@@ -103,4 +114,113 @@ public void resolveDependencies() {
 //    public BallerinaToml ballerinaToml() {
 //        return this.packageContext.ballerinaToml();
 //    }
+
+
+    /** Returns an instance of the Package.Modifier.
+     *
+     * @return  module modifier
+     */
+    public Modifier modify() {
+        return new Modifier(this);
+    }
+
+    private static class ModuleIterable implements Iterable {
+        private final Collection<Module> moduleList;
+
+        public ModuleIterable(Collection<Module> moduleList) {
+            this.moduleList = moduleList;
+        }
+
+        @Override
+        public Iterator<Module> iterator() {
+            return this.moduleList.iterator();
+        }
+
+        @Override
+        public Spliterator spliterator() {
+            return this.moduleList.spliterator();
+        }
+    }
+
+    /**
+     * Inner class that handles package modifications.
+     */
+    public static class Modifier {
```

Comment:
Follow the same technique as : https://github.com/ballerina-platform/ballerina-lang/pull/26009/files#r493393568

Reply:
Fixed in 9a7d6fe